pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v1/Namespaces.java

```diff
@@ -575,7 +575,19 @@ public BookieAffinityGroupData getBookieAffinityGroup(@PathParam("property") Str
         validateNamespaceName(property, cluster, namespace);
         return internalGetBookieAffinityGroup();
     }
-    
+
+    @DELETE
+    @Path("/{property}/{cluster}/{namespace}/persistence/bookieAffinity")
+    @ApiOperation(hidden = true, value = "Delete the bookie-affinity-group from namespace-local policy.")
+    @ApiResponses(value = { @ApiResponse(code = 403, message = "Don't have admin permission"),
```

Comment:
Add a period.

Please check all similar cases in this PR.

Reply:
I think we generally don't add period in response message  and all the apis are having similar pattern.